src/rcc.rs

```diff
@@ -261,6 +262,7 @@ impl CFGR {
         }
 
         // set prescalers and clock source
+        #[cfg(not(feature = "stm32f100"))]
```

Comment:
Maybe it makes more sense to explicitly specify the boards which things do work on, i.e. replace `not(feature = "stm32f100")` with `feature = "stm32f103"`. Though I suppose this depends on how specific these things are to different boards.

@therealprof @TeXitoi Any thoughts on this? How is it done in the other HALs?

Reply:
In other hal impls we're doing it exactly like you suggest, however cleaning that up shouldn't be a big deal. But now that you mention it, there's another thing missing... CI.

Reply:
Makes sense. Updated.